packages/react-image/src/components/Image/useImage.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-ignore Typings require esModuleInterop
+import { objectFitImages } from 'object-fit-images';
+import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utils';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { ImageProps, ImageState } from './Image.types';
+
+const mergeProps = makeMergeProps();
+
+const isFitSupported = (function() {
+  const testImg = new Image();
+
+  return 'object-fit' in testImg.style && 'object-position' in testImg.style;
+})();
+
+/**
+ * Given user props, returns state and render function for an Image.
+ */
+export const useImage = (props: ImageProps, ref: React.Ref<HTMLElement>, defaultProps?: ImageProps): ImageState => {
+  const resolvedRef = useMergedRefs(ref, React.useRef(null));
+  const state = mergeProps(
+    {
+      ref: resolvedRef,
+      as: 'img',
+    },
+    defaultProps,
+    resolveShorthandProps(props, []),
+  );
+
+  React.useEffect(() => {
+    if (!isFitSupported) {
+      objectFitImages(state.ref.current);
+    }
+  }, [state]);
```

Comment:
```suggestion
    },
    // objectFitImages() should be executed once per element
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [],
  );
```

Looks like a merge issue, but it's different from current `master`